### PR TITLE
added kdelante14's solution to learn_ruby project

### PR DESCRIPTION
**Pull request by kdelante14**
```
This is a PR template, If you are adding a solution link to the curriculum leave this as is. If you are not, then delete it and write the message you wish.
Thank you,
The Odin Project team.

```

### CODE REVIEW

File: web_development_101/project_ruby.md [hunk 1]
```diff
@@ -26,8 +26,12 @@ This exercise will involve a lot of figuring things out.  There aren't very deta
 
 * ["Official" Test-First Ruby solutions](https://github.com/ultrasaurus/test-first-teaching/tree/master/learn_ruby) (look for the "solutions" folder inside each exercise)
 * Add your solution below this line!
+<<<<<<< HEAD
```
Comment (arun1595) [inline web_development_101/project_ruby.md]: Please remove this line.

File: web_development_101/project_ruby.md [hunk 2]
```diff
@@ -26,8 +26,12 @@ This exercise will involve a lot of figuring things out.  There aren't very deta
...
+* [kdelante14's solutions](https://github.com/kdelante14/learn_ruby)
+=======
```
Comment (arun1595) [inline web_development_101/project_ruby.md]: And this too.

File: web_development_101/project_ruby.md [hunk 3]
```diff
@@ -26,8 +26,12 @@ This exercise will involve a lot of figuring things out.  There aren't very deta
...
 * [Adong520's solutions](https://github.com/Adong520/learn_ruby)
 * [Tarelochkin's solutions](https://github.com/Tarelochkin/learn_ruby)
+>>>>>>> 3ed995ded35ae9d9efd7cd9476b9d3219493f679
```
Comment (arun1595) [inline web_development_101/project_ruby.md]: And this too.

**Comment by kdelante14** [comment]
Removed the lines.

File: web_development_101/project_ruby.md [hunk 4]
```diff
@@ -26,9 +26,12 @@ This exercise will involve a lot of figuring things out.  There aren't very deta
 
 * ["Official" Test-First Ruby solutions](https://github.com/ultrasaurus/test-first-teaching/tree/master/learn_ruby) (look for the "solutions" folder inside each exercise)
 * Add your solution below this line!
+* [kdelante14's solutions](https://github.com/kdelante14/learn_ruby)
 * [Webdev-burd's soltions](https://github.com/webdev-burd/learn_ruby)
+
```
Comment (arun1595) [inline web_development_101/project_ruby.md]: Why is this newline present? Please remove it.

File: web_development_101/project_ruby.md [hunk 5]
```diff
@@ -26,9 +26,12 @@ This exercise will involve a lot of figuring things out.  There aren't very deta
...
 * [Adong520's solutions](https://github.com/Adong520/learn_ruby)
 * [Tarelochkin's solutions](https://github.com/Tarelochkin/learn_ruby)
+
```
Comment (arun1595) [inline web_development_101/project_ruby.md]: And this one too.

**Pull request merged**